Configuration/AlCa/python/autoCond.py

```diff
@@ -86,6 +86,9 @@
 from Configuration.HLT.autoCondHLT import autoCondHLT
 autoCond = autoCondHLT(autoCond)
 
+from Configuration.AlCa.autoCondPhase2 import autoCondPhase2
+autoCond = autoCondPhase2(autoCond)
```

Comment:
an explanatory comment should be added here indicating that autoCond keys "phase2_realistic_T*" are provided

Reply:
done